libs/seo/router/src/schema/collect-schema.ts

```diff
@@ -0,0 +1,8 @@
+import { ActivatedRouteSnapshot } from "@angular/router";
+import { collect } from "@daffodil/core";
+import { Schema } from "@daffodil/seo";
+
+export const collectSchemaFromRoutes = (route: ActivatedRouteSnapshot): Schema<unknown> => {
+	const routes = collect(route, (r) => r.children);
+	return [...new Set(routes.map((r) => r.data?.daffSeoData?.schema).filter((el) => el))];
```

Comment:
Note that this function is unfortunately odd due to https://angular.dev/api/router/RouterConfigOptions#paramsInheritanceStrategy causing multiple instances of the same schema to be merged together.